src/main/java/net/kyori/adventure/text/minimessage/transformation/inbuild/RainbowTransformation.java

```diff
@@ -79,10 +80,17 @@ public void load(final String name, final List<TagPart> args) {
     super.load(name, args);
 
     if (args.size() == 1) {
-      try {
-        this.phase = Integer.parseInt(args.get(0).value());
-      } catch (final NumberFormatException ex) {
-        throw new ParsingException("Expected phase, got " + args.get(0), this.argTokenArray());
+      String value = args.get(0).value();
+      if (args.get(0).value().startsWith("^")) {
+        this.reversed = true;
+        value = value.replace("^", "");
```

Comment:
Reference a `static final String REVERSE` constant in this class instead, please.